as/cNFT/assembly/models/persistent_tokens.ts

```diff
@@ -100,11 +100,22 @@ export class PersistentTokens {
         return token
     }
 
+    /**
+     * @param tokenId: Id of the token to remove
+     * @param accountId: Account of token to remove
+     */
+    remove_token(tokenId: TokenId, accountId: AccountId): void {
+        this._tmap.delete(tokenId);
+
+        this.remove(tokenId, accountId);
```

Comment:
I believe this should be called from `burn_design` better

Reply:
Then it will call two functions from `core.ts` to remove. We can just copy the same logic from remove function and paste it instead of `this.remove()`. What do you think ?

Hey @devey, What do you think about this?